tests/tests_integration_pytest.yml

```
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Empty play to gather facts
+  hosts: all
+
+- name: Run the role once first
+  hosts: all
+  roles:
+    - linux-system-roles.network
+
+- import_playbook: playbooks/centos7_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version >= '7'
+
+- import_playbook: playbooks/fedora32_deps.yml
+  when:
+    - ansible_distribution == "Fedora"
+    - ansible_distribution_major_version == '32'
+
+- name: Run Pytest tests
+  hosts: all
+  vars:
+    - rundir: /run/system-roles-test
+  tasks:
+    - file:
+        state: directory
+        path: /run
+
+    - command: git rev-parse --show-toplevel
+      register: testdir
```

```suggestion
      register: git_top_directory
```

It seems that `git_top_directory` or similar is a better name here than `testdir` since the actual tests are in a sub directory.